src/elements/content-sidebar/activity-feed/comment/types.js

```diff
@@ -12,3 +18,10 @@ export type OnCommentEdit = (args: {
     status?: FeedItemStatus,
     text?: string,
 }) => void;
+
+export type OnCommentStatusChange = (args: {
+    hasMention?: boolean,
```

Comment:
This was done to make the types work correctly. Refactor so types work more intentionally, potentially deduplicating `OnAnnotationStatusChange` and `OnCommentStatusChange`